src/TypeScript/NodeJSLocator/Resources.resx

```diff
@@ -117,6 +117,12 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="INFO_NotFoundInGlobalPath" xml:space="preserve">
+    <value>[NodeJsLocator] Could not find node.exe in PATH environment variable</value>
```

Comment:
Technically this message is partial, as we search in path AND in the program files, but I think it's enough for the user. We _would_ want them to update the path var.